packages/orchestration/src/cosmos-api.ts

```diff
@@ -36,7 +36,7 @@ export type CosmosValidatorAddress = ChainAddress & {
 };
 
 /** Represents an IBC Connection between two chains, which can contain multiple Channels. */
-export type IBCConnectionInfo = {
+export interface IBCConnectionInfo {
```

Comment:
I'd expect to use interfaces for things with methods, but this is an ordinary data structure. Why not leave it as a type alias?

Reply:
I guess being able to refer to each property with a link is nice.